### DISCUSSION TITLE
a question about "time"

### DISCUSSION THREAD

**Discussion by qianbenhyu** [projectdiscovery/nuclei#4257] [Q&A] [answered]
There are some deserialization vulnerabilities that can take few seconds to exploit
_(The exploitation of vulnerabilities takes a few seconds for files to land)._ 
I hope there is a simple way to define "timeout" and "sleep" for multiple HTTP requests (sending the second request after 10 seconds after the first request) in ".yaml".

**Comment by MetzinAround** [comment, marked as answer]
https://docs.nuclei.sh/template-guide/code

You might be able to use the new code protocol to write a script that puts a sleep request in. Other than that I think there's a DSL helper called `wait_for(seconds)` that waits in seconds the amount of time you input. 

Remember, though, that time based stuff can be tricky as there's a lot of factors that go into something taking a certain amount of time.
